themes/powerline-multiline/powerline-multiline.theme.bash

```diff
@@ -129,7 +130,12 @@ function powerline_scm_prompt {
 }
 
 function powerline_cwd_prompt {
-    CWD_PROMPT="$(set_rgb_color - ${CWD_THEME_PROMPT_COLOR}) \w ${normal}$(set_rgb_color ${CWD_THEME_PROMPT_COLOR} -)${normal}$(set_rgb_color ${CWD_THEME_PROMPT_COLOR} -)${THEME_PROMPT_SEPARATOR}${normal}"
+    CWD_PROMPT_COLOR=${CWD_THEME_PROMPT_COLOR}
+    CAN_I_RUN_SUDO=$(sudo -n uptime 2>&1 | grep "load" | wc -l)
+    if [ ${CAN_I_RUN_SUDO} -gt 0 ]; then
```

Comment:
I actually tried that and it didn't work on the Mac. It looks like `sudo` is always returning 0, regardless of its state:
![image](https://cloud.githubusercontent.com/assets/907063/7204589/d2fd5798-e523-11e4-9e7b-8a7fc70593b9.png)

Using the return code would be easier/nicer, but it doesn't seem to work.